src/SpaceTime/IBackedBy.cs

```diff
@@ -0,0 +1,17 @@
+﻿
+namespace Bearded.Utilities.SpaceTime
+{
+    /// <summary>
+    /// Represents a type that is backed by a specific singular numeric type value.
+    /// </summary>
+    /// <typeparam name="T">The type of the backing value.</typeparam>
+    public interface IBackedBy<out T>
+        where T : struct
+    {
+        /// <summary>
+        /// Returns the numeric value of the type.
+        /// </summary>
+        T NumericValue { get; }
+    }
+
+}
```

Comment:
I understand. I think that's a very interesting thought, but I am not sure if an interface is the right tool. Actually, what this interface does is exposing implementation (the name very much implies to do so at least), and with implementing the interface you commit to keeping it that way. I suppose you could always rename the interface to something like `ReducesTo<T>` and make the `NumericValue` differently named as well that does not imply implementation.

Even then, the interface seems to be there to make a system in another library easier. The question is whether that is worth committing to an interface to.